Backend/dev/migrations/rider-app/1149-add-new-messages-for-yatri-sathi.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE atlas_app.merchant_service_usage_config ADD COLUMN enable_dashboard_sms Boolean;
+UPDATE atlas_app.merchant_service_usage_config SET enable_dashboard_sms=false;
+ALTER TABLE atlas_app.merchant_service_usage_config ALTER COLUMN enable_dashboard_sms SET NOT NULL;
+
+UPDATE atlas_app.merchant_service_usage_config SET enable_dashboard_sms = true
+  WHERE merchant_id = 'da4e23a5-3ce6-4c37-8b9b-41377c3c1a51'; -- change merchant id to merchant id of yatri sathi
+
+INSERT INTO atlas_app.merchant_message (merchant_id, message_key, message) VALUES
+    ('da4e23a5-3ce6-4c37-8b9b-41377c3c1a51', 'SEND_DASHBOARD_OTP',
+'Dear User,
```

Comment:
We need to get this messages approved from TRAI, are they approved?

Reply:
Approved